Header header
float32 gnss_x
float32 gnss_y
float32 gnss_z
int32 gnss_flag
int32 capacity
int32 current
int64 tem
int32 moveFlag